feature/tabsflow/moreflow/donate/src/main/java/com/epmedu/animeal/tabs/more/donate/presentation/viewmodel/DonateState.kt

```diff
@@ -0,0 +1,9 @@
+package com.epmedu.animeal.tabs.more.donate.presentation.viewmodel
+
+import com.epmedu.animeal.tabs.more.donate.domain.DonateInformation
+
+data class DonateState(
+    val donationInformation: List<DonateInformation> = emptyList(),
```

Comment:
should be ImmutableList

Reply:
@egorikftp  What do you mean? We have List and MutableList in the Kotlin standard library

Reply:
Please search in repository, for compose we use immutable collections

Reply:
Fixed